Brain/Brain.ino

```diff
@@ -82,8 +88,12 @@ void loop() {
       i++;
```

Comment:
Is the command array defined somewhere first? I don't see that happening anywhere

Reply:
line 26